xl-op/blueprint.yaml

```
@@ -82,18 +73,18 @@ spec:
       ignoreIfSkipped: true
       overrideDefault: true
       promptIf: !expr ServerType == 'dai-release' && (ProcessType == 'install' || ProcessType == 'upgrade')
-      prompt: "Enter the image name (eg: <imageName> from <repositoryName>/<imageName>:<tagName>):"
+      prompt: "Enter the application image name (eg: <imageName> from <repositoryName>/<imageName>:<tagName>):"
```

```suggestion
      prompt: "Enter the release server image name (eg: <imageName> from <repositoryName>/<imageName>:<tagName>):"
```
since this is only for release image, can we have it similar to other fields like for deploy or deploy task engine image.

---

These changes are for now to see the difference between remote runner questions and the application questions. If you think that we need to additionally distinct prompts, can you please create a new ticket